hack/generator/pkg/astmodel/external_package_reference.go

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT license.
+ */
+
+package astmodel
+
+import (
+	"fmt"
+	"strings"
+)
+
+// ExternalPackageReference indicates a library package that needs to be imported
```

Comment:
```suggestion
// ExternalPackageReference indicates a package to be imported from an external source, such as github or the go standard library.
```
(or some other text that removes "library")

Reply:
Revised wording is much clearer, thanks.